tests/subsys/fs/nffs_fs_api/src/nffs_test_utils.h

```diff
@@ -0,0 +1,103 @@
+/*
```

Comment:
any reason why you are including the full license text?

Reply:
most files in tests/ are files from NFFS repository adjusted to use with Zephyr so I left licenses intact (like for a NFFS code pull-in) - just added spdx